cogs/public.py

```diff
@@ -42,12 +50,19 @@ async def serverinfo(self, ctx: commands.Context):
         raise NotImplementedError('Command requires implementation and permission set-up.')
     
     @commands.hybrid_command(aliases=['ui'])
-    async def userinfo(self, ctx: commands.Context):
+    async def userinfo(self, ctx: commands.Context, member: discord.Member = None):
```

Comment:
Great start on this feature - but you might want to add a couple more fields. Refer to the relevant code snippet in [the original IB.ai's implementation of this command](https://github.com/ib-ai/IB.ai/blob/729ff5c5201aa30dcc96dd1e0f19a486ef314466/src/main/java/com/ibdiscord/command/registrar/RegistrarUtil.java#L358).